src/Serilog/Core/Logger.cs

```diff
@@ -88,6 +88,9 @@ public sealed class Logger : ILogger, ILogEventSink, IDisposable
             _enricher = enricher;
         }
 
+        internal bool HasOverrides => _overrideMap != null;
+        internal LevelOverrideMap OverrideMap => _overrideMap;
```

Comment:
Could we use the `bool TryGetOverrideMap(out OverrideMap)` pattern to avoid potentially exposing a null-valued property?